.eslintrc.js

```diff
@@ -1,11 +1,22 @@
 module.exports = {
   extends: ['algolia', 'algolia/react'],
   rules: {
-    'react/prop-types': 0,
-    'no-param-reassign': 0,
     'eslint-comments/disable-enable-pair': 0,
+    // Do not enforce a specific import order
+    'import/order': 0,
```

Comment:
Why not?

Reply:
I wanted to prevent this PR from being too difficult to follow. I can auto fix these rules (I'm thinking this one and the other you commented on: `react/function-component-definition` / `react/jsx-handler-names`) in a separate PR. Would that be ok?

Reply:
Yes let's absolutely do this in a separate PR.